webui/react/src/pages/WorkspaceList.tsx

```diff
@@ -178,7 +178,7 @@ const WorkspaceList: React.FC = () => {
         dataIndex: 'name',
         defaultWidth: DEFAULT_COLUMN_WIDTHS['name'],
         key: V1GetWorkspacesRequestSortBy.NAME,
-        onCell: onRightClickableCell,
+        onCell: () => ({ ...onRightClickableCell(), 'data-testid': 'name' }),
```

Comment:
actually, i like `data-test-value` on variable test hooks. nbd either way

Reply:
think I'll keep this as is for now just because it's already the convention in `UserManagement.tsx`